### PR TITLE
[MIRROR] Changes felinid diet to be more like a cat's and less like a human's

### PR DESCRIPTION
**Pull request by AustationBot**
Original PR: https://github.com/BeeStation/BeeStation-Hornet/pull/2445
--------------------

## About The Pull Request
Makes felinids like meat and dairy, makes them dislike vegetables and sugar, and makes them immune to carpotoxin (so they can eat fish without being slightly annoyed by a very baby sauce toxin). They're also now neutral to gross and raw foods, which lets them eat mice and raw meats.

NSV codelads port this or I uh (vague but clearly joking threat IDk im tired)

## Why It's Good For The Game
Makes a playable species slightly less identical to human

## Changelog
:cl:
add: Felinids are now immune to carpotoxin and can have little a fish
tweak: Felinids like meat and dairy, and dislike vegetables and sugar.
/:cl:

### CODE REVIEW

**Comment by coolsurf6** [comment]
But can it have little salami?

**Comment by FatCat919** [comment]
Add anti-freeze chem, and make sure cats need to drink from a bowl >:)

**Comment by coolsurf6** [comment]
Remember @FatCat919 this is just a mirror so it has already been added to bee... But god damn would that be a fucking meme. 

Section off the entire station or the employee rooms to have fucking bowls in them, and the only way to eat is from a bowl. That is the cringiest fucking thing ever but puts felinids in their place if they wanna roleplay as ooga booga cat anime ear person

File: code/modules/reagents/chemistry/reagents/toxin_reagents.dm [hunk 1]
```diff
@@ -142,6 +142,11 @@
 	toxpwr = 2
 	taste_description = "fish"
 
+/datum/reagent/toxin/carpotoxin/on_mob_metabolize(mob/living/carbon/L)
+	if(iscatperson(L))
+		toxpwr = 0
+	..()
+
```
Comment (MCterra10) [inline code/modules/reagents/chemistry/reagents/toxin_reagents.dm:149]: felinid buff but actually kinda based?
oh god I'm connnflicting arghhh

**Pull request merged**